numpy/core/src/multiarray/typeinfo.c

```diff
@@ -105,10 +104,36 @@ PyArray_typeinforanged(
     return entry;
 }
 
-void typeinfo_init_structsequences(void)
+/* Backport, only needed here */
+#if PY_VERSION_HEX < 0x03040000
+    static int
+    PyStructSequence_InitType2(PyTypeObject *type, PyStructSequence_Desc *desc) {
+        PyStructSequence_InitType(type, desc);
+        if (PyErr_Occurred()) {
+            return -1;
+        }
+        return 0;
+    }
+#endif
```

Comment:
Needed only because I think we should backport this to 1.16

Reply:
We dropped support for Python < 3.4 in 1.12, but this doesn't hurt.

Reply:
2.7 < 3.4 too. I made exactly the same mistake while writing this.

Reply:
Oh, duh :)